src/test/java/core/basesyntax/service/operationwithfruits/PurchaseOperationHandlerTest.java

```diff
@@ -0,0 +1,72 @@
+package core.basesyntax.service.operationwithfruits;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.service.FruitTransaction;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class PurchaseOperationHandlerTest {
```

Comment:
Same thing too many positive methods. To know that a method works correctly, one is enough. Fix everywhere